docs/providers/prometheus_remote_write.md

```diff
@@ -0,0 +1,22 @@
+# Prometheus Remote Write
+
+## Exporter
+
+The `prometheus_remote_write` exporter allows to export SLO metrics to [Prometheus](https://prometheus.io) via the remote write API.
+
+```yaml
+exporters:
+  prometheus_remote_write:
+    url: ${PROMETHEUS_REMOTE_WRITE_ENDPOINT} # <https://<prometheus>/api/v1/push
+```
+
+Optional fields:
+  * `metrics`: List of metrics to export ([see docs](../shared/metrics.md)).
+  * `username`: Username for Basic Auth.
+  * `password`: Password for Basic Auth.
+  * `job`: Name of `RemoteWrite` job. Defaults to `slo-generator`.
```

Comment:
@oktocat I think you didn't set the job label, but I think is a nice to have (since the other prometheus exporter has this option), is up to you to keep or not this option